dataGenerator/src/main/java/com/filip/datagenerator/Service/Impl/DataGeneratorImpl.java

```diff
@@ -0,0 +1,60 @@
+package com.filip.datagenerator.Service.Impl;
+
+import com.filip.datagenerator.Enum.Status;
+import com.filip.datagenerator.KafkaSenders;
+import com.filip.datagenerator.Model.Account;
+import com.filip.datagenerator.Model.Transaction;
+import com.filip.datagenerator.Service.DataGenerator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+
+@Service
+public class DataGeneratorImpl implements DataGenerator {
+
+    @Value("${OFFLINE_TRANSACTION_UPPER_LIMIT}")
+    Integer limit;
+
+    @Autowired
+    KafkaSenders sender;
+
+    @Override
+    public void insertAccounts() throws ExecutionException {
+        int n = 0;
+        while (n < 21) { // TODO: 10001
+            List<Transaction> tlist = new ArrayList<>();
+            Random random = new Random(505000);
+            double amount = 15000 + random.nextDouble();
+            Account a = new Account(UUID.randomUUID(), UUID.randomUUID(), amount, tlist, true);
+            sender.sendAccount(a);
+            n++;
+        }
+    }
+
+    @Override
+    public void generateOfflineTransaction() throws InterruptedException {
+
+        int n = 0;
+        while (n != 3) {
+            Transaction transaction = new Transaction(UUID.randomUUID(),
+                    UUID.randomUUID(), null, 0.0, Status.WAITING);
+            n++;
+            sender.sendTransaction(transaction);
+        }
+        Thread.sleep(2000); // TODO: 500
```

Comment:
Make this configurable

Reply:
Logic here is questionable - why are you generating accountIds randomly since you had a task to create accounts with already defined accountIds? Why is every single transaction's amount is zero? it means that there is no money to be transacted, right?

Reply:
Its configurable now, logic was explained on wednesday. Will change logic completely to use inserted accounts.